app/src/main/res/layout-land/profile_progress_header.xml

```diff
@@ -24,7 +24,6 @@
       android:id="@+id/profile_edit_image"
       android:layout_width="160dp"
       android:layout_height="160dp"
-      android:background="@color/color_def_white_light"
```

Comment:
This background is not effecting anything.

Reply:
I have cross checked it. You can remove it.

Reply:
> I have cross checked it. You can remove it.

Okay sure. Thanks.